docs/setup-cluster/deploy-cluster/slurm/singularity.rst

```diff
@@ -0,0 +1,247 @@
+:orphan:
+
+#################################
+ Provide a Container Image Cache
+#################################
+
+When the cluster does not have Internet access or if you want to provide a local cache of container
+images to improve performance, you can download the desired container images to a shared directory
+and then reference them using file system paths instead of Docker registry references.
+
+There are two mechanisms you can use to reference cached container images depending upon the
+container runtime in use.
+
+-  :ref:`referencing-local-image-paths`
+-  :ref:`singularity-image-cache`
+-  :ref:`manage-singularity-cache`
+-  :ref:`manage-enroot-cache`
+
+***********************
+ Default Docker Images
+***********************
+
+Each version of Determined utilizes specifically-tagged Docker containers. The image tags referenced
+by default in this version of Determined are described below.
+
++-------------+--------------------------------------------------------------------------+
+| Environment | File Name                                                                |
++=============+==========================================================================+
+| CPUs        | ``determinedai/environments:py-3.8-pytorch-1.12-tf-2.11-cpu-2b7e2a1``    |
```

Comment:
We only match parse the following 3 lines my matching the image type in column 1 and getting the path from column 2.   

```
| CPUs        | ``determinedai/environments:py-3.8-pytorch-1.12-tf-2.11-cpu-6eceaca``    |
| NVIDIA GPUs | ``determinedai/environments:cuda-11.3-pytorch-1.12-tf-2.11-gpu-6eceaca`` |
| AMD GPUs    | ``determinedai/environments:rocm-5.0-pytorch-1.10-tf-2.7-rocm-6eceaca``  |
```

Reply:
we may have to manually remove the duplicate result from Algolia